src/05-derivatives/01-introduction.md

```diff
@@ -127,10 +127,10 @@ share the following (non-required) ones:
 | Description    | RECOMMENDED. Free-form natural language description of the nature of the file.                                                                                                                                                                                                                                                                        |
 | Sources        | OPTIONAL. A list of paths relative to dataset root pointing to the file(s) that were directly used in the creation of this derivative. For example in a chain of A->B->C, “C” should only list “B” as Sources, and “B” should only list “A” as Sources. However in case X and Y jointly contribute to Z, then “Z” should list “X” and “Y” as Sources. |
 | RawSources     | OPTIONAL. A list of paths relative to dataset root pointing to the BIDS-Raw file(s) that were used in the creation of this derivative.                                                                                                                                                                                                                |  |
-| ReferenceMap   | REQUIRED when a custom template or file is used. A path to a file that was used as, or can be used as, a reference image for determining the coordinate space of this file. The path should start with a `/` and should be relative to the root of the dataset.                                                                                       |
-| ReferenceIndex | REQUIRED when an index into a 4D (ReferenceMap or NonstandardReference) file is used. Used to index into a 4D spatial-reference file.                                                                                                                                                                                                                 |
+| Reference   | REQUIRED when a custom template (i.e., [not listed as a default template](## Template Based Coordinate Systems)) or any other appropriate file was used as reference. When set, this field contains the path to the file that was used as, or can be used as, a reference image, surface or sphere for determining the coordinate system and space of this file. The path should start with a `/` and should be relative to the root of the dataset.                                                                                       |
+| ReferenceIndex | REQUIRED when an index into a 4D (`Reference` or `NonstandardReference`) file is used. Used to index into a time-varying spatial-reference file.                                                                                                                                                                                                                 |
```

Comment:
Shouldn't `NonstandardReference` be removed?

Reply:
Probably yes, I hadn't been following this up. This was more of a correctness issue. WDYT @satra?